usage-viewer/README.md

```diff
@@ -0,0 +1,62 @@
+# Retool Usage Viewer - Identify top apps by pageviews and queries
+
+## Why you need this
+As an administrator of a self-hosted Retool instance, you want to understand the most frequently used apps in your instance. The Retool audit log contains the information you need, but it's difficult to access and make sense of.
+
+The Retool Usage Viewer makes your job easier by displaying app usage in a way that's easy to consume.
+
+## What does the Usage Viewer do?
+Usage Viewer is a ready-made app that lets you do these things:
+
+1. View all applications in your Retool instance, sorted by pageviews or queries
+2. Filter by app name, folder, and time range
+3. Identify top apps visually using built-in charts
+
+## Screenshots - TODO
+
+## Technical details and setup
+The Usage Viewer connects to the Postgres instance containing your [Retool storage database](https://docs.retool.com/docs/configuring-retools-storage-database), where a copy of all audit logs are stored. This database is named `hammerhead_production` and is only accessible when self-hosting Retool.
+
+### Create a read-only database user
+In order to use the Usage Viewer, first create a Postgres user with **read-only** access to the following tables in the `hammerhead_production` database:
+
+- `pages` (contains information about Retool apps)
+- `folders`
+- `audit_trail_events`
+
+To create a user and set a password, use the following command:
+
+```
+CREATE USER read_only_user WITH PASSWORD 'secure_password'
+```
+
+To assign the required permissions, connect to `hammerhead_production` and use the following commands:
+
+```
+GRANT CONNECT ON DATABASE hammerhead_production TO read_only_user;
+GRANT USAGE ON SCHEMA public TO read_only_user;
+GRANT SELECT ON pages TO read_only_user;
+GRANT SELECT ON folders TO read_only_user;
+GRANT SELECT ON audit_trail_events TO read_only_user;
+```
+
+## Create a Postgres resource connection
+In the Retool dashboard, [create a new Postgres resource](https://docs.retool.com/docs/postgresql-integration) and provide the necessary connection details.
```

Comment:
Retool user interface?

Reply:
Worth calling out the Resource Name on creation needs to be exactly the same?

Reply:
I like "dashboard" personally. Not sure if we have a standard convention for referring to it though.

I added a section to the docs about wiring up the queries to the correct resource once you've imported the app, since it seems unlikely people will choose the same name each time : )

Reply:
I also think "dashboard" is a bit overloaded when we are talking about Retool. 
One alternative: "the Resources page in Retool"?

I'd second Bryan too on providing an exact name of the Resource that you want them to create, instead of asking them to "connect app queries" -- it will feel like a more seamless setup! And since the Resource here is so specific—with the read-only user and all, the name can be something like "Retool Audit Log Reader".